software/chartdb.yml

```diff
@@ -0,0 +1,16 @@
+name: ChartDB
+website_url: https://chartdb.io
+description: Database diagrams editor for visualizing and designing databases with a single query, including AI-powered DDL script export in multiple SQL dialects.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database Management
+  - Analytics
+  - Software Development - IDE & Tools
+source_code_url: https://github.com/chartdb/chartdb
+demo_url: https://app.chartdb.io/examples
+stargazers_count: 3833
+updated_at: "2024-10-27"
+archived: false
```

Comment:
```suggestion
```
This will be maintained automatically, no need for these lines 👍🏻

Reply:
Oops. tnx, removing it now.

Reply:
@kokomo123 fixed that. any other issues? 
Thank you for your help!

Reply:
One last thing. Unfortunately, it seems there are no tagged releases. Rather then pull from main/latest all the time, users should be able to verify that its stable and have changelogs, etc. We can check back in around 4 months from the first stable tagged release.

Reply:
@kokomo123 Thank you for pointing that out! I didn’t realize the importance of having tagged releases for stability and changelogs. I’ll make adjustments to remove the related updates to tags and focus solely on adding ChartDB in this PR, which should help streamline the review process. Would it be possible to push this quickly?

Reply:
Unfortunately, it is not possible to push through PRs quickly (as far as im aware of according to the requirements/guidelines). Usually if you start making tagged releases now and we wait 4 months, we can then reopen the PR, and promptly approve it and merge it within a week.

Reply:
@kokomo123 what? let's close this PR then, and I will open a new PR without a tagged releases, why to wait 4 months now?

Reply:
I'll have @nodiscc @Ki-er explain it better as they may have a better explaination then what i'm saying.

Reply:
@kokomo123 Any update on that? :)

Reply:
I just added the "standard" reply to this frequent issue here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#no-tagged-releases